src/Analyzers/Core/Analyzers/AddAccessibilityModifiers/AbstractAddAccessibilityModifiersDiagnosticAnalyzer.cs

```diff
@@ -39,5 +52,25 @@ private void AnalyzeTree(SyntaxTreeAnalysisContext context, CompilationOptions c
         ProcessCompilationUnit(context, option, (TCompilationUnitSyntax)context.Tree.GetRoot(context.CancellationToken));
     }
 
-    protected abstract void ProcessCompilationUnit(SyntaxTreeAnalysisContext context, CodeStyleOption2<AccessibilityModifiersRequired> option, TCompilationUnitSyntax compilationUnitSyntax);
+    protected void CheckMemberAndReportDiagnostic(
+        SyntaxTreeAnalysisContext context,
+        CodeStyleOption2<AccessibilityModifiersRequired> option,
+        SyntaxNode member)
+    {
+        if (!this.AddAccessibilityModifiers.ShouldUpdateAccessibilityModifier(
+                this.AccessibilityFacts, member, option.Value, out var name, out var modifiersAdded))
```

Comment:
Strange indenting here.

Reply:
i indent one more from the parent invocation.  that way you can tell arguments from sibling clauses.